test/controllers/user_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class UserControllerTest < ActionDispatch::IntegrationTest
```

Comment:
Why 2 UserControllerTest?
UserControllerTest
 UsersControllerTest

Reply:
I added it by mistake